checks/cs50/2017/x/speller/checks.py

```diff
@@ -0,0 +1,58 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Speller(Checks):
+
+    @check()
+    def exists(self):
+        """dictionary.c, dictionary.h, and Makefile exist"""
+        super(Speller, self).exists("dictionary.c")
+        super(Speller, self).exists("dictionary.h")
+        super(Speller, self).exists("Makefile")
+
+    @check("exists")
+    def compiles(self):
+        """speller compiles"""
+        self.include("speller.c")
```

Comment:
Hm, `require` could be assumed (by TFs) to mean "student must have a file called `speller.c` is all. I keep mentioning `copy` partly because that's how Docker does this; it's (essentially) a synonym for `add` in Docker too. How about that, `add`? Pretty unambiguous?